### DISCUSSION TITLE
The organization selection is not being displayed on the login screen.

### DISCUSSION THREAD

**Discussion by kelvinsleonardo** [keycloak/keycloak#35581] [Q&A] [answered]
I registered a user and linked them to two organizations. However, after accessing the login screen, there is no option to select an organization. Is there any additional configuration required ?

**Comment by kelvinsleonardo** [comment, marked as answer]
Found the solution. It is necessary to include scope=organization in the login screen parameters.

**Comment by mildlypranav** [comment]
Hey — we ran into the same gap and built an open-source Keycloak Authenticator SPI that handles this:

**https://github.com/revspot/keycloak-select-org-authenticator**

It plugs into the browser authentication flow as a post-login step and queries the user's **native Keycloak 26+ Organization memberships**:
- 0 orgs → denies access
- 1 org → auto-selects silently
- 2+ orgs → shows a picker page

Stores the selection in session notes (`kc.org`) so the built-in Organization Membership token mapper picks it up automatically. Hope it helps!
